airflow/providers/dbt/cloud/example_dags/example_dbt_cloud.py

```diff
@@ -22,7 +22,7 @@
 try:
     from airflow.operators.empty import EmptyOperator as DummyOperator
 except ModuleNotFoundError:
-    from airflow.operators.dummy import DummyOperator
+    from airflow.operators.dummy import DummyOperator  # type: ignore
```

Comment:
I know its a slightly bigger change, but shouldn't we do this as

```suggestion
    from airflow.operators.dummy import DummyOperator as EmptyOperator  # type: ignore
```

and then change all the uses to `EmptyOperator()`?

Reply:
doesn't sound like a big change I can do this.
But users for Airflow<2.3 are not really aware of `EmptyOperator`. It might be strange to them?

Reply:
This is fine. We want to promote EmptyOperator usage and import Dummy as Emptyi in import on "module not found" makes a nice fallback. We only need falback for compatibilty but we want that anyone looking at our providers from now on used Empty*.

The tests will fail if they don't do the fallback so we will detect it - we can also even add a e specialized pre-commit that will look for this:


and suggest changing:

```
from airflow.operators.empty import EmptyOperator
```
into 

```
try:
    from airflow.operators.empty import EmptyOperator
except ModuleNotFoundError:
    from airflow.operators.dummy import DummyOperator as Empty Operator
```

if import is detected without try/ except around it. That's easy check.

Reply:
But we can do it as next step.

Reply:
it's a small change no worries :) 
see https://github.com/apache/airflow/pull/22974/commits/321fc1fa02c4ca03a98d336dffc848385b6aa7ee